src/client/layouts/ConsentsLayout.tsx

```diff
@@ -77,14 +77,16 @@ export const ConsentsLayout: FunctionComponent<ConsentsLayoutProps> = ({
         </main>
         <ConsentsBlueBackground>
           <div css={[gridItem(gridItemColumnConsents), controls]}>
-            <Button
-              iconSide="right"
-              nudgeIcon={true}
-              icon={<SvgArrowRightStraight />}
-              type="submit"
-            >
-              Save and continue
-            </Button>
+            {!error && (
```

Comment:
i'm worried this could mean that users get stuck on this page if there is an error?

Reply:
Previously the button posted to `/consents/` which doesn't exist.